src/components/cards/LunchCard.vue

```diff
@@ -5,34 +5,99 @@
           No Lunch Data
          <what-is-this>Stevenson.Space relies on quality data dispayed by Stevenson's lunch website. We will promptly restore lunch functionality once data is provided there.</what-is-this>
     </div>
-    <div v-else v-for="(items, name) in lunch" :key="name" class="lunch">
-      <div class="name">{{ name }}</div>
-      <div v-for="item in items" :key="item" class="item">
-        {{ item }}
+    <div v-else>
+      <div v-if="isMorning" class="lunch">
+        <div class="name">Ala Carte Breakfast</div>
+        <div v-for="item in alBreakfast" :key="item" class="item">
+          {{ item }}
+        </div>
+      </div>
+      <div v-for="(items, name) in lunch" :key="name" class="lunch">
+        <div class="name">{{ name }}</div>
+        <div v-for="item in items" :key="item" class="item">
+          {{ item }}
+        </div>
+      </div>
+      <div v-for="(items, name) in mData" :key="name" class="exLunch">
+        <div class="name">{{ name }}</div>
+        <div v-for="item in items" :key="item" class="item">
+          {{ item }}
+        </div>
       </div>
     </div>
+    <p>{{isExpanded}}</p>
+    <div class="arrow-container">
+      <font-awesome-icon
+        v-show="(expandNums < 3)"
+        :icon="icons.faChevronDown"
+        class="expand arrow"
+        @mousedown="expand"
+      />
+      <font-awesome-icon
+        v-show="(expandNums > 0)"
+        :icon="icons.faChevronUp"
+        class="collapse arrow"
+        @mousedown="condense"
+      />
+    </div>
   </card>
 </template>
 
 <script>
+import { faChevronDown, faChevronUp } from '@fortawesome/free-solid-svg-icons';
 import getLunch from '@/utils/lunch';
 import Card from '@/components/Card.vue';
 import WhatIsThis from '@/components/WhatIsThis.vue';
 import { mapState } from 'pinia';
 import useScheduleStore from '@/stores/schedules';
+import miscData from '@/data/miscLunch.json';
 
 export default {
   components: { Card, WhatIsThis },
   data() {
     return {
       noLunchData: false,
+      icons: {
+        faChevronDown,
+        faChevronUp,
+      },
+      expandNums: 0,
+      alBreakfast: ['Bacon', 'Sausage', 'Potatoes', 'Eggs'],
     };
   },
   computed: {
     ...mapState(useScheduleStore, ['bell', 'date']),
     lunch() {
       return getLunch(this.date);
     },
+    isMorning() {
+      const currentDate = new Date();
```

Comment:
I think we should be able to use our date/time function in schedules.ts, it can be accessed using     `...mapState(useScheduleStore, ['date'])` as a computed variable in a component. Do a search for that line of code to see how it's used. Therefore we won't have to regenerate the date.